importer/main.py

```diff
@@ -302,6 +303,24 @@ def location_extras(resource, payload_string):
         del obj["resource"]["type"]
         payload_string = json.dumps(obj, indent=4)
 
+    try:
+        if len(adminLevel) > 0:
```

Comment:
if the csv has no value and is assigned a "0" in the try/except above, then the assignment here will always be a 0
lets have it get into the else to check the parent instead